diff --git a/modules/codec/videotoolbox.m b/modules/codec/videotoolbox.m
--- a/modules/codec/videotoolbox.m
+++ b/modules/codec/videotoolbox.m
@@ -38,6 +38,10 @@
 #import <Foundation/Foundation.h>
 #import <TargetConditionals.h>
 
+#ifndef TARGET_OS_MACCATALYST
+#define TARGET_OS_MACCATALYST 0
+#endif
+
 #import <sys/types.h>
 #import <sys/sysctl.h>
 #import <mach/machine.h>
@@ -1251,7 +1255,7 @@ static int StartVideoToolbox(decoder_t *p_dec)
         return VLC_EGENERIC;
     }
 
-#if !TARGET_OS_IPHONE
+#if !TARGET_OS_IPHONE || TARGET_OS_MACCATALYST
     CFDictionarySetValue(destinationPixelBufferAttributes,
                          kCVPixelBufferIOSurfaceOpenGLTextureCompatibilityKey,
                          kCFBooleanTrue);